packages/venia-concept/src/components/Header/header.css

```diff
@@ -56,3 +56,66 @@
     grid-template-columns: 1fr 1fr;
     justify-items: end;
 }
+
+.searchFallback {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 0.5rem;
+    padding: 0 1rem;
+}
+
+.input {
+    composes: input from '../TextInput/textInput.css';
+    max-width: 24rem;
+}
+
+.loader,
+.loader:before,
+.loader:after {
+    --dot-size: 2em;
+    --dot-font-size: 6px;
+    --dot-shadow-offset: calc(-1 * var(--dot-size) + var(--dot-font-size));
+    border-radius: 50%;
+    width: var(--dot-size);
+    height: var(--dot-size);
+    animation-fill-mode: both;
+    animation: pulse 1.8s infinite ease-in-out;
+}
+
+.loader {
+    color: rgb(var(--venia-grey-dark));
+    font-size: var(--dot-font-size);
+    margin: var(--dot-shadow-offset) auto 0;
+    position: relative;
+    transform: translateZ(0);
+    animation-delay: -0.16s;
+}
+
+.loader:before,
+.loader:after {
+    content: '';
+    position: absolute;
+    top: 0;
+}
+
+.loader:before {
+    color: rgb(var(--venia-grey));
```

Comment:
Not to nitpick but on my screen this is super light and really difficult to see. If this value is from the mocks, fine, but maybe it needs to be darker.